app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -29,6 +29,22 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_group_volumes
+    labels = [_("Name"), _("Property"), _("Value")]
+    h = {:labels => labels}
+    h[:values] = []
+    @record.container_volumes.each do |volume|
+      is_first = true
+      (volume.attributes.keys - %w(id container_group_id type name)).each do |key|
+        h[:values] << [
+          is_first ? volume.name : nil,
+          key,
```

Comment:
@zakiva I think the *Properties* should be more friendly. E.g. keep a dictionary somewhere for:

    {
        :git_repository => "Git Repository",
        ...
        :common_read_only => "Read-only",
        ...
    }

@h-kataria any suggestion on where to keep it? Should it be possible to localize it, etc.